### PR TITLE
Replace uptime robot with hund

### PR DESCRIPTION
**Pull request by MacFan4000**

### CODE REVIEW

File: zones/miraheze.wiki [hunk 1]
```diff
@@ -27,9 +27,8 @@ $ORIGIN miraheze.wiki.
 ; Servers
 
 ; Services
-status		               CNAME	stats.uptimerobot.com.
 phab		               CNAME	phabricator.miraheze.org.
-status3.miraheze.wiki.         CNAME   miraheze.hund.io.
+status                         CNAME    miraheze.hund.io.
```
Comment (paladox) [inline zones/miraheze.wiki]: keep it as status.miraheze.wiki i think that fixed the cert error anyways.

Comment (JohnFLewis) [reply zones/miraheze.wiki]: really? something that has zero impact fixed it?

File: zones/miraheze.wiki [hunk 2]
```diff
@@ -27,9 +27,8 @@ $ORIGIN miraheze.wiki.
 ; Servers
 
 ; Services
-status		               CNAME	stats.uptimerobot.com.
 phab		               CNAME	phabricator.miraheze.org.
-status3.miraheze.wiki.         CNAME   miraheze.hund.io.
+status.miraheze.wiki           CNAME    miraheze.hund.io.
```
Comment (paladox) [inline zones/miraheze.wiki]: needs to have the . at the end
so like this

status.miraheze.wiki.           CNAME    miraheze.hund.io.

Comment (JohnFLewis) [reply zones/miraheze.wiki]: or just status per me above :)

**Pull request merged**